Docs/Card/card.html

```diff
@@ -0,0 +1,220 @@
+          </div>
+          <div class="card__price pad-8-16 d-flex">
+            <div class="card__price--final">Rs:25000</div>
+            <div class="card__price--original"><strike>40000</strike></div>
+            <div class="card__price--offer">37.5% off</div>
+          </div>
+          <div class="card__footer pad-8-16">
+            <div class="card-footer__div flex-sp-bw">
+              <button class="btn btn-primary btn-wrapper">
+                <div class="btn_icon"><i class="fa-solid fa-cart-shopping"></i></div>
+                <div class="btn_content">Add to Cart</div>
+              </button>
+              <button class="btn btn-outline btn-wrapper">
+                <div class="btn_icon"><i class="fa-solid fa-heart"></i></div>
+                <div class="btn_content">Wishlist</div>
+              </button>
+            </div>
+          </div>
+          <div class="badge card__badge">New</div>
+       </div>
+     </div>
+      
+     <h3 class="content_h3">Card with Dismiss :</h3>
+     <div class="card-container flex-center">
+      <div class="card flex-column">
+        <div class="card__img"><img src="/Assets/Sofa.jpg" alt="Sofa"></div>
+        <div class="card__info pad-8-16">
+          <p class="card-title">Sofa's & More</p>
+          <p class="card-desc">Seater sofa - Grey</p>
+        </div>
+        <div class="card__price pad-8-16 d-flex">
+          <div class="card__price--final">Rs:25000</div>
+          <div class="card__price--original"><strike>40000</strike></div>
+          <div class="card__price--offer">37.5% off</div>
+        </div>
+        <div class="card__footer pad-8-16">
+          <div class="card-footer__div flex-sp-bw">
+            <button class="btn btn-primary btn-wrapper">
+              <div class="btn_icon"><i class="fa-solid fa-cart-shopping"></i></div>
+              <div class="btn_content">Add to Cart</div>
+            </button>
+            <button class="btn btn-outline btn-wrapper">
+              <div class="btn_icon"><i class="fa-solid fa-heart"></i></div>
+              <div class="btn_content">Wishlist</div>
+            </button>
+          </div>
+        </div>
+        <div class="card__dismiss"><i class="fa-solid fa-circle-xmark"></i></div>
+      </div>
+     </div>
+
+     <h3 class="content_h3">Card with Text Overlay :</h3>
+     <div class="card-container flex-center">
+      <div class="card flex-column">
+        <div class="card__img"><img src="/Assets/Sofa.jpg" alt="Sofa"></div>
+        <div class="card__info pad-8-16">
+          <p class="card-title">Sofa's & More</p>
+          <p class="card-desc">Seater sofa - Grey</p>
+        </div>
+        <div class="card__price pad-8-16 d-flex">
+          <div class="card__price--final">Rs:25000</div>
+          <div class="card__price--original"><strike>40000</strike></div>
+          <div class="card__price--offer">37.5% off</div>
+        </div>
+        <div class="card__footer pad-8-16">
```

Comment:
you can name the class little better. pad-8-16 is little confusing.

Reply:
Thanks for the review Vaibhav, Sure I will try to make it more clear. Here it meant top & bottom 8px and let & right 16px